frontend/src/index.css

```
@@ -55,3 +54,29 @@
     }
   }
 }
+
+/** Loader **/
+
+.cls-1 {
```

```suggestion
.loader-svg path {
```

Please give this a proper semantic name.